### DISCUSSION TITLE
Skip request using contains value if request has dynamic id with some static text

### DISCUSSION THREAD

**Discussion by tvtester** [DannyDainton/newman-reporter-htmlextra#397] [Ideas]
Right now if we want to skip some request from report, we need to provide full request name. Then only those are skipped from the report. however this is problematic when request name has dynamic Id with some static/predefined text. In this case cannot pass the full request name.

so it would be good to have capability where if we give partial text to match and those matching one get skipped.

**Comment by DannyDainton** [comment]
PRs are always welcome. 😁

This is an open source project so if you'd like to try and implement that, I am more than happy to review your work.
